export async function registrationError(catchError) {
  console.log('OPS! Error with registration', catchError);
}
